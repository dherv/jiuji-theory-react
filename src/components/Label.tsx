import styled from "styled-components";

const Label = styled.label`
  font-weight: bold;
  font-size: 0.75rem;
`;

export default Label;
