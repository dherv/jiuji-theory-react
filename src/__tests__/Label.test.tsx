import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import Label from "../components/Label";

test("should display a label with the text", () => {
  render(<Label>test</Label>);
  expect(screen.getByText("test")).toBeInTheDocument();
});

test("should render the label correctly", () => {
  const { asFragment } = render(<Label>test</Label>);
  expect(asFragment()).toMatchSnapshot();
});
