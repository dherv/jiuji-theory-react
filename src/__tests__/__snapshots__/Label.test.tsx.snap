// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the label correctly 1`] = `
<DocumentFragment>
  <label
    class="sc-AxjAm ggatId"
  >
    test
  </label>
</DocumentFragment>
`;
